protos/node_reputation/node_reputation.proto

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+package nodereputation;
+
+service NodeReputation {
+    rpc NodeReputation(NodeQuery) returns (NodeReputationRecord) {}
+    rpc UpdateReputation(NodeUpdate) returns (UpdateReply) {}
+    rpc FilterNodeReputation(NodeFilter) returns (NodeReputationRecords) {}
+    rpc PruneNodeReputation(NodeQuery) returns (UpdateReply) {}
+}
+
+message NodeReputationRecord {
+    string source = 1;
```

Comment:
source?

okay i guess these are all supposed to be kind of node reputation events? i think i'm understanding why there's PruneNodeReputation and stuff. i think we should probably only ever have one row per node in the db and update the existing row instead of adding a new one. that way auditSuccess and auditFail would be totals, we could change uptime to be pingSuccess and pingFail, etc